packages/voice/src/app/agent/page.tsx

```diff
@@ -248,11 +352,15 @@ class ChatManager {
     } else if (final) {
       const request = this.pendingRequests[normalized];
       request.active = true;
-      this.tts.play(request.outMessage);
-      if (!request.done) {
-        this.onOutputChange?.(request.outMessage, false, request.requestLatency!);
+      if (!request.didStart) {
```

Comment:
I think I'd probably handle this by just not starting requests for partial transcrips if we're using Fixie, then we can control this with a single if statement